src/test/java/step2/CalculationsTest.java

```diff
@@ -0,0 +1,82 @@
+package step2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CalculationsTest {
+
+    @DisplayName("덧셈 테스트")
+    @ParameterizedTest
+    @ValueSource(strings ={"2 + 3 + 7"})
+    void plusTest(String value){
+        String[] result = value.split(" ");
+        assertThat(calc(result)).isEqualTo(12);
+
+    }
+
+    @DisplayName("뺄셈 테스트")
+    @ParameterizedTest
+    @ValueSource(strings ={"2 - 3"})
+    void minusTest(String value){
+        String[] result = value.split(" ");
+        assertThat(calc(result)).isEqualTo(-1);
+    }
+
+    @DisplayName("곱셈 테스트")
+    @ParameterizedTest
+    @ValueSource(strings ={"2 * 3"})
+    void multiplyTest(String value){
+        String[] result = value.split(" ");
+        assertThat(calc(result)).isEqualTo(6);
+    }
+
+    @DisplayName("나눗셈 테스트")
+    @ParameterizedTest
+    @ValueSource(strings ={"10 / 3"})
+    void divideTest(String value){
+        String[] result = value.split(" ");
+        assertThat(calc(result)).isEqualTo(3);
+    }
+
+    @DisplayName("사칙 연산을 모두 포함하는 기능 구현")
+    @ParameterizedTest
+    @ValueSource(strings ={"10 + 20 * 30 / 10"})
+    void allTest(String value){
+        String[] result = value.split(" ");
+        assertThat(calc(result)).isEqualTo(90);
+    }
+
+    @DisplayName("존재하지 않는 연산자 IllegalArgumentException")
+    @ParameterizedTest
+    @ValueSource(strings ={"2 3"})
+    void illegalArgumentException(String value){
+        String[] result = value.split(" ");
+        assertThatExceptionOfType(IllegalArgumentException.class)
+        .isThrownBy(() ->{
+            assertThat(calc(result));
+        });
```

Comment:
```suggestion
        assertThatThrownBy(() -> calc(result))
            .isInstanceOf(IllegalArgumentException.class);
```
저는 수행 실체가 앞에오고 기대값이 뒤에오는 위 형태를 선호합니다. 
또다른 방법으로 `llegalArgumentException` 의 경우 미리 만들어진 단언도 있어요. 
```java
        assertThatIllegalArgumentException().isThrownBy(() -> {
			calc(result);
        });
```
비교해 보시죠 😀